### PR TITLE
Support cgroup filtering for libbpf-tools

### PR DESCRIPTION
**Pull request by chethanah**
- Previously in #3741 and #3890 - cgroup2 based filtering for libbpf-tools `biolatency` and `biosnoop` was supported
- In this PR support more tools for cgroup2 based filtering.

Kindly review and feedback if any changes required for merge.

### CODE REVIEW

File: libbpf-tools/cpudist.c [hunk 1]
```diff
@@ -17,6 +18,8 @@
 static struct env {
 	time_t interval;
 	pid_t pid;
+	char	*cgroupspath;
```
Comment (chenhengqi) [inline libbpf-tools/cpudist.c:21]: nit: Use space after `char`. Same for some other tools. Please follow the original code.

Otherwise looks good to me.

Comment (chethanah) [reply libbpf-tools/cpudist.c]: Thank you @chenhengqi for the review.
I've updated code to match original format.

**Review by chenhengqi — APPROVED** [review]
LGTM

**Comment by yonghong-song** [comment]
@chethanah could you rebase on top of master. The patch has some conflicts with latest master.

**Comment by yonghong-song** [comment]
@chethanah could you rebase on top of master. The patch has some conflicts with latest master.

**Comment by chethanah** [comment]
> @chethanah could you rebase on top of master. The patch has some conflicts with latest master.

@yonghong-song, I've rabased changes to latest. Kindly verify.

**Review by davemarchevsky — APPROVED** [review]

**Pull request merged**